build/sdns/local.mog

```diff
@@ -0,0 +1,26 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+
+# Copyright 2024 Guo-Rong Koh
+
+license LICENSE license=MIT
+
+dir path=$(CONFIG) owner=$(USER) group=$(GROUP) mode=0700
+dir path=$(DATA) owner=$(USER) group=$(GROUP) mode=0700
+dir path=var/log/$(PREFIX)/$(PROG) owner=$(USER) group=$(GROUP) mode=0755
+
+group groupname=$(PROG) gid=5353
+user ftpuser=false username=$(PROG) uid=5353 group=$(PROG) \
```

Comment:
system UIDs should be below `100` to avoid clashes with user IDs. have a look at `doc/idlist.md` and look for a free uid/gid pair (e.g. `58`). and document the ID you are using in idlist.md.